### ISSUE TITLE
Immersion usb controller

### ISSUE THREAD

**Issue by mikedeluca** [timschumi/iforce-binary-driver#2]
Hi, I have an arcade racing cabinet which uses an immersion pcb and a happ arcade force feedback wheel. The immersion drivers are from 2009, and are 64 bit and do install on win11, but if I run any 64 bit game that initializes force feedback, the game crashes with a fault in the immersion physical driver DLL. I tried the driver above, but I get an error 10 when trying to load the driver in device manager. I’ve also tried the T150 driver from thrustmaster, same issue. Drivers and board info here: http://forum.arcadecontrols.com/index.php/topic,135310.0.html

**Comment by timschumi** [comment]
I'm afraid that this doesn't give me much to work with, "Error 10" appears to just be a generic error code for "something went wrong while initializing the device". It's also very unlikely that this driver will just work with any random Immersion-based board, since the configuration still has to be adapted to the specific device.

**Comment by mikedeluca** [comment]
Yeah, unfortunately I was afraid of that. I've reached out to Immersion to see if they have anything newer (or at least that doesn't crash on 64 bit), No clue if I'll hear back.

**Comment by timschumi** [comment]
Let me know in case they do, if whatever they have can be public I'd love to try it out as well (not that I'm currently plagued by crashes, everything I tried so far either works or it simply does nothing).

**Comment by MircoRed** [comment]
Hello, I got to work Wingman Formula Force with Saitek drivers with your inf mod with Windows 10 64bits and Force feedback effects in 64 bits Games, not only 32bits games.
Thanks to @WallyCZ in https://github.com/WallyCZ/saitek-cyborg-ff I downloaded SaiQFFB5.dll and rename to SaiQFF04.dll. Then, I replace this renamed dll on the folder I downloaded from your repository. After, I installed drivers. Also you can copy the new SaiQFF04.dll in Windows/system32/ if you have already installed the original repository version.
I tested Asseto Corsa Competizzione in 32bits and 64bits with FFB, BeamNG.drive with FFB and Forza Horizon 5 with FFB using Forza EmuWheel. At the moment, no issues.
I want to thank everyone who made this possible with his work.

**Comment by Invictaz** [comment]
@MircoRed You mean this older wheel?

![image](https://github.com/timschumi/iforce-binary-driver/assets/20338545/49d08a13-7df4-4f92-a55c-e8100996b13f)

**Comment by MircoRed** [comment]
Yes

**Comment by Invictaz** [comment]
> Yes

Great to have it working again.
